docs/ivys.js

```diff
@@ -16,6 +16,53 @@ function IvyApp($grid) {
     this.$menuArrow.click(this.toggleFooterMenu);
 }
 
+IvyApp.prototype.initFirebase = function () {
+    "use strict";
+    // Shortcuts to Firebase SDK features.
+    this.auth = firebase.auth();
+    this.storage = firebase.storage();
+    this.storageRef = this.storage.ref();
+    
+    // Logs debugging information to the console.
+    firebase.database.enableLogging(false);
+    
+    // Initiates Firebase auth and listen to auth state changes.
+    this.auth.onAuthStateChanged(this.onAuthStateChanged.bind(this));
+
+};
+
+IvyApp.prototype.signIn = function () {
+    // Sign a user in anonymously
+    "use strict";
+    
+    this.auth.signInAnonymously().catch(function (error) {
+        var errorCode = error.code,
+            errorMessage = error.message;
+        console.error("Error signing in " + errorMessage);
+        return errorCode;
```

Comment:
Well, I don't need to do anything with the promise. The user just signs in anonymously and that's it... Later if the user starts picking favorites then we'd store that on the user's anonymous account.